reviewer-topics.json

```diff
@@ -189,8 +189,9 @@
   {
     "github_handle": "dagurtomas",
     "zulip_handle": "@**Dagur Asgeirsson**",
-    "top_level": ["t-condensed", "t-category-theory", "t-algebra", "t-algebraic-geometry", "t-algebraic-topology", "t-topology"],
+    "top_level": ["t-condensed", "t-category-theory", "t-algebraic-topology", "t-algebraic-geometry", "t-algebra", "t-topology"],
```

Comment:
I don't know if the order matters here, but if the bot somehow prioritizes according to order, this is approximately the correct order

Reply:
The assignment algorithm doesn't take the ordering of areas into account yet, but that's a good idea for a future feature!